### ISSUE TITLE
Email notification: Remind users to complete their profile

### ISSUE THREAD

**Issue by mandily-p** [BCcampus/eypd#606]
### User story
As a site owner, 
I want to remind users to complete their profiles and fill in their certification expiry date
Because I want them to benefit from all features of the website, and I need that data to report back to the ministry.

### Summary
While another issue exists to prompt users _within_ the site to update their profiles and certification expiry dates, we also need another way to prompt users who have not recently visited the website. This will also serve as a reminder to use the site to manage their PD.

### Acceptance criteria
If a user has not completed both their profile and filled in their certification expiry date...
- Send a friendly reminder email at 3 different times
  - 24 hours after account creation
  - 1 week after account creation
  - 1 month after account creation
- Email should use a branded site template including a logo
- Email should contain a link back to the user's profile page and/or myEYPD page so that they can easily access what we're promoting
- Email should contain a link for users to opt out of notifications (either directly or through the notification settings on the site)
- If a user is not logged in, they should be first taken to the login modal, and then the page associated with the original link they clicked.

### Copy
@kathreenriel Can you add the copy needed for each email or links to another doc that has the copy?

### Related/Blocked by/Blocking
Related - #602 - Persistent messaging

**Comment by dshaykewich** [comment]
Won't fix.
